src/modules/wallet/api/utils/getWalletNameLabel.ts

```diff
@@ -0,0 +1,12 @@
+import { WalletType } from '../types'
+
+export function getWalletNameLabel(walletType: WalletType): string {
```

Comment:
Should we make it as `Record<WalletType, string>` ?

Reply:
And I would name it as `WalletTypeLabel`

Reply:
It ends up being supplied as walletName fallback. Do you think it should be a separate field then?

Reply:
Actually, it's not a big deal, you can ignore it :) 
I suggested it just to avoid ambiguity, because in fact it's not a wallet name (it's a fallback of wallet name)

Reply:
Ah! Makes sense. I'll need to update some parts of the code to clean up namings, can include it there.